e2e/portalicious/tests/ViewPayment/ValidateViewPaymentsTable.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { test } from '@playwright/test';
+import { format } from 'date-fns';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import NLRCProgram from '@121-service/src/seed-data/program/program-nlrc-ocw.json';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import {
+  programIdOCW,
+  registrationsOCW,
+} from '@121-service/test/registrations/pagination/pagination-data';
+
+import TableComponent from '@121-e2e/portalicious/components/TableComponent';
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import PaymentsPage from '@121-e2e/portalicious/pages/PaymentsPage';
+
+const registrationIds = ['Reg. #1', 'Reg. #2', 'Reg. #3', 'Reg. #4', 'Reg. #5'];
+const ascedingRegistrationIds = [...registrationIds].sort((a, b) =>
+  a.localeCompare(b),
+);
+const descedingRegistrationIds = [...registrationIds].sort((b, a) =>
+  a.localeCompare(b),
+);
+
+const names = [
+  'Anna Hello',
+  'John Smith',
+  'Lars Larsson',
+  'Luiz Garcia',
+  'Sophia Johnson',
+];
+
+const ascedingNames = [...names].sort((a, b) => a.localeCompare(b));
+const descedingNames = [...names].sort((b, a) => a.localeCompare(b));
```

Comment:
```suggestion
const descedingNames = [...names].sort((a, b) => b.localeCompare(a));
```

Same comment/logic as above.